docs/DeveloperGuide.md

```diff
@@ -268,30 +268,101 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `PlaNus` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case 01: Add a task**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to add a specific person to the list.
 
-    Use case ends.
+2. PlaNus adds the person.
+
+   Use case ends.
+
+**Extensions**
+
+- 1a. The given index is invalid.
+
+  - 1a1. AddressBook shows an error message.
+
+    Use case resumes at step 1.
+
+
+
+**Use case 02: Delete a task**
+
+**MSS**
+
+1. User requests to list persons.
+
+2. PlaNus shows a list of persons.
+
+3. User requests to delete a specific person in the list.
+
+4. PlaNus deletes the person.
+
+   Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+- 2a. The list is empty.
```

Comment:
are we going to skip displaying or just display an empty list? If we are just displaying an empty list, is it still considered an extension?

Reply:
added extension